### ISSUE TITLE
repo.continuum.io timing out?

### ISSUE THREAD

**Issue by estatematt** [conda/conda#3598]
Please consider posting the following information to the
conda GitHub issue tracker at:

```
https://github.com/conda/conda/issues
```

Current conda install:

```
           platform : osx-64
      conda version : 4.2.9
   conda is private : False
  conda-env version : 4.2.9
conda-build version : 2.0.2
     python version : 2.7.12.final.0
   requests version : 2.11.1
   root environment : /Users/matthewaidekman/anaconda  (writable)
default environment : /Users/matthewaidekman/anaconda
   envs directories : /Users/matthewaidekman/anaconda/envs
      package cache : /Users/matthewaidekman/anaconda/pkgs
       channel URLs : https://repo.continuum.io/pkgs/free/osx-64/
                      https://repo.continuum.io/pkgs/free/noarch/
                      https://repo.continuum.io/pkgs/pro/osx-64/
                      https://repo.continuum.io/pkgs/pro/noarch/
        config file : None
       offline mode : False
```

`$ /Users/matthewaidekman/anaconda/bin/conda create -n python3 python=3.5 anaconda`

```
Traceback (most recent call last):
  File "/Users/matthewaidekman/anaconda/lib/python2.7/site-packages/conda/exceptions.py", line 473, in conda_exception_handler
    return_value = func(*args, **kwargs)
  File "/Users/matthewaidekman/anaconda/lib/python2.7/site-packages/conda/cli/main.py", line 144, in _main
    exit_code = args.func(args, p)
  File "/Users/matthewaidekman/anaconda/lib/python2.7/site-packages/conda/cli/main_create.py", line 68, in execute
    install(args, parser, 'create')
  File "/Users/matthewaidekman/anaconda/lib/python2.7/site-packages/conda/cli/install.py", line 420, in install
    raise CondaRuntimeError('RuntimeError: %s' % e)
CondaRuntimeError: Runtime error: RuntimeError: Runtime error: Could not open u'/Users/matthewaidekman/anaconda/pkgs/pyzmq-15.4.0-py35_0.tar.bz2.part' for writing (HTTPSConnectionPool(host='repo.continuum.io', port=443): Read timed out.).
```

**Comment by kalefranz** [comment]
From reading through the full stack trace, I'm not convinced the problem is with repo.continuum.io. If you've retried a few times, has the problem persisted?

**Comment by estatematt** [comment]
Switched wifi networks and it worked.  WEIRD.

**Comment by mcd0well** [comment]
Also getting this error out of nowhere? There is always a chance this is down to a corp proxy but it seems that others (myself included) are getting the same error: #3601

**Comment by kalefranz** [comment]
Intermittent HTTP error, likely due to flakey internet connection.  Retrying will probably resolve the issue.  Conda 4.3 has longer default timeouts that are also configurable.
